couchdb-cluster-playground/steps.py

```diff
@@ -0,0 +1,152 @@
+import os
+import subprocess
+import time
+import threading
+
+from couchdbkit import Server
+import requests
+
+import lib.couchdb_custom as couch_info
+
+
+NODES = [
+    {
+        'url': 'http://localhost:5984',
+        'docker_name': 'server-0'
+    },
+    {
+        'url': 'http://localhost:15984',
+        'docker_name': 'server-1'
+    },
+    {
+        'url': 'http://localhost:25984',
+        'docker_name': 'server-2'
+    },
+]
+
+should_create_docs = False  # Boolean used for doc creation thread
+
+
+def create_docs(reporting_interval=4):
+    def create_docs_thead():
+        # thread that runs and inserts documents
+        total_count = 0
+        last_reported_time = 0
+        new_count = 0
+
+        while should_create_docs:
+            doc = {"mydoc": "test"}
```

Comment:
It may be useful to try creating larger documents. Since we're hypothesizing that there is a bug in Couch it's possible it is able to easily keep up with tiny documents, but would have a harder time with larger ones.

Reply:
I thought about this too, but figured many tiny docs was harder. But yeah I can put in a random length string.